src/fhirtypes/CodeSystem.ts

```diff
@@ -3,8 +3,8 @@ import { Meta } from './specialTypes';
 import { Extension } from '../fshtypes';
 import { Narrative, Resource, Identifier, CodeableConcept, Coding } from './dataTypes';
 import { ContactDetail, UsageContext } from './metaDataTypes';
-import { HasName, HasId } from './common';
-import { applyMixins } from '../utils';
+import { HasName, HasId } from './mixins';
+import { applyMixins } from '../utils/Mixin';
```

Comment:
We figured this would be likely to happen. One way or another, we'll have to resolve that merge conflict when it appears, but I don't think it'll be too difficult to do that.